e2e/report-user.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { test, expect } from '@playwright/test';
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+// To run this test you will need to run the email server.
+// To do this: https://contribute.freecodecamp.org/#/how-to-catch-outgoing-emails-locally?id=using-mailhog
+
+test('should be possible to report a user from their profile page', async ({
+  page
+}) => {
+  await page.goto('/twaha');
+
+  // The following line is required if you're running the test in local development
+  await page.getByRole('button', { name: 'Preview custom 404 page' }).click();
```

Comment:
```suggestion
  // If you build the client locally, delete the button click below.
  if (!process.env.CI) {
    await page.getByRole('button', { name: 'Preview custom 404 page' }).click();
  }
```
As a rough heuristic, most people working locally will be running the client in dev mode, so this should be right more often than not.